### PR TITLE
add post processing script to make static analyzer report smaller

### PR DESCRIPTION
**Pull request by gartung**

### CODE REVIEW

**Comment by smuzaffar** [comment]
@gartung , I would suggest to keep this script `postprocess-scan-build.py` in cmssw (along with the others in Utilities/StaticAnalyzers/scripts

File: run_clang_static_analysis [hunk 1]
```diff
@@ -23,6 +23,11 @@ cp -pv $LOCALRT/tmp/bloom.bin $LOCALRT/src/Utilities/StaticAnalyzers/scripts/blo
 
 mv $LOCALRT/llvm-analysis/* $WORKSPACE/llvm-analysis
 chmod -R a+r $WORKSPACE/llvm-analysis
+cp -p postprocess-scan-build.py $WORKSPACE/llvm-analysis
+cd $WORKSPACE/llvm-analysis
+move index.html index_full.html
+./postprocess-scan-build.py >index.html 2>&1
```
Comment (smuzaffar) [inline run_clang_static_analysis]: this will fail unless postprocess-scan-build.py becomes part of cmssw/Utilities/StaticAnalyzers/scripts

**Comment by gartung** [comment]
Move to CMSSW pull request.

**Comment by gartung** [comment]
Needs https://github.com/cms-sw/cmssw/pull/24984

**Comment by smuzaffar** [comment]
this can go in now as we run postprocess-scan-build.py only if exists

**Pull request merged**